src/test/java/se/sundsvall/smloader/service/scheduler/DatabaseCleanerSchedulerTest.java

```diff
@@ -25,7 +26,7 @@ class DatabaseCleanerSchedulerTest {
 	@Test
 	void executeWithEntitiesToRemove() {
 		scheduler.execute();
-		verify(databaseCleanerServiceMock).cleanDatabase(any(OffsetDateTime.class));
+		verify(databaseCleanerServiceMock).cleanDatabase(any(OffsetDateTime.class), anyString());
```

Comment:
Use ErrandConstants.MUNICIPALITY_ID instead of anyString()?
(this is the value used in the DatabaseCleanerScheduler class)